marlowe-test/src/Spec/Marlowe/Plutus/Specification.hs

```diff
@@ -400,11 +400,12 @@ checkValueOutput =
   let
     modifyAfter =
       do
+        delta <- lift $ oneof [chooseInteger (-5, -1), chooseInteger (1, 5), arbitrary `suchThat` (/= 0)]  -- Ensure small non-zero integers.
         let
-          -- Add one lovelace to the output to the script.
+          -- Add or subtract some lovelace to the output to the script.
           incrementOwnOutput txOut@(TxOut address value _ _)
-            | address == semanticsAddress = txOut {txOutValue = value <> singleton adaSymbol adaToken 1}
-            | otherwise                  = txOut
+            | address == semanticsAddress = txOut {txOutValue = value <> singleton adaSymbol adaToken delta}
```

Comment:
what is this for?

Reply:
I generalized the test so that it didn't just perturb by 1 lovelace, but used other small and large integers, too.